docker-compose/one-node-ray-syntho-ui/docker-compose.yaml

```diff
@@ -92,9 +92,9 @@ services:
         condition: service_healthy
     healthcheck:
       test: wget --no-verbose --tries=1 -O /dev/null http://backend:8000/api/docs/ || exit 1
-      interval: 5s
-      timeout: 1s
-      retries: 2
+      interval: 30s
+      timeout: 2s
+      retries: 10
```

Comment:
That is a lot of retries with 30 seconds in between. Should that be either less between the intervals or less retries?

Reply:
I still use the old images, and backend takes too much time. I don't want to make any changes when testing the existing functionality not to introduce any side effect (cuz then I will need to retest every single scenario). And with the current image set and resource usages, this healthcheck config was the one the deployment was successful.  I am running on a minimal setup and trying to finalize it based on this that 
--
I don't think this numbers are a major issue considering the timeout check on the CLI. Cuz any of those images can't be ran successfully if the duration is too much. 

--
please let me know if you have an objection or serious concerns for something I didn't consider for. So we can fine-tune it accordingly.

Reply:
Could we not just adjust them to the same interval as the Core API? I'm not necessarily worried about the startup time of the application, more the time it takes now to mark it unhealthy. In this particular case, if anything happened to the backend, it would take 300 seconds before it gets marked as unhealthy, which is a long time for it to be downed (which a restart might fix, but that will also only happen after 300 seconds. If it's startup time that has an impact on the CLI, should we then not adjust `start_period` instead?

Reply:
For now, I'll accept the PR (as it's been open for a while), but I'll open a new PR so we can discuss about those intervals.